### PR TITLE
Sync with thehyve repo

### PR DESCRIPTION
**Pull request by inodb**

### CODE REVIEW

**Comment by pieterlukasse** [comment]
@inodb this is just a fast-forward right?

**Review by pieterlukasse — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,47 +1,63 @@
-# cbioportal-docker
+# cbioportal-docker @ The Hyve
```
Comment (pieterlukasse) [inline README.md:1]: :)

Comment (fedde-s) [reply README.md:1]: I wrote that heading and first paragraph to explain how the approach differs from that documented in the [cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal/blob/v1.10.1/Dockerfile) repo itself. If we're making this approach a community effort but maintaining two separate workarounds for the limitations in the cBioPortal build process, we should probably rephrase this.

**Comment by jjgao** [comment]
Is this PR just for tracking or we would like to merge it?

**Comment by fedde-s** [comment]
I'm not sure what the purpose of this repository is; I feel like it's not being used. If the plan is to mirror or move the setup that we're maintaining at The Hyve to the cBioPortal organisation, then I suppose it should be merged.

**Comment by fedde-s** [comment]
However, since cBioPortal/cbioportal#3505 makes it unnecessary for`portal.properties` to be passed through Maven, it might be possible to tweak this Dockerfile a bit and solve the drawbacks of both the The Hyve setup and the one documented in the cBioPortal manual. Then we could merge this into the main cBioPortal repository, and produce Docker images that work for everyone.

**Comment by jjgao** [comment]
@fedde-s @inodb on a high level, I think we should main one docker repo and maintain it together. As we are figuring out moving things to cloud using docker and kubernetes, I hope this will happen.

**Comment by fedde-s** [comment]
The future I imagine is one Dockerfile in the frontend repo, one in the backend repo, and a docker-compose file to deploy a full stack in the `cBioPortal/cbioportal` repo, along with documentation.

**Comment by jjgao** [comment]
@fedde-s maybe we should have an assembly or cloud repo?

**Comment by fedde-s** [comment]
I suppose that could work, as long as it's clearly documented in `cBioPortal/cbioportal`. That's where I expect people who want to install and start contributing to the software to search for information on the stack as a whole, even before they know that the frontend and backend are maintained in separate repositories.

**Comment by pieterlukasse** [comment]
@inodb @jjgao should we merge this one or close it?

**Comment by fedde-s** [comment]
I'm still occasionally spending some time figuring out how we can make the entrypoints for all cBioPortal scripts properly accept runtime configuration so that we can unify the Dockerfiles, but in the meantime I don't see the point of this repository.

**Comment by inodb** [comment]
@fedde-s That's great to hear! I think one solution might be to push two images to docker hub. One that doesn't recompile when running it and when that does.

**Comment by fedde-s** [comment]
The issue with that is that cBioPortal still can't seem to be configured without recompiling the whole Java layer, to get the import layers for study and side data working. So far, every Dockerised deployment needs its own image, which hard-wires things such as the database password.

**Comment by fedde-s** [comment]
Once that resolved, I feel like there would no longer be a need for a container that does recompile all Java code before every single command, nor for configuring anything but the defaults at image build time. We could get the best of both approaches, and make an flexibly orchestratable image.

**Pull request merged**